integration-tests/src/test/java/oracle/weblogic/kubernetes/ItElasticLoggingFluentd.java

```diff
@@ -280,11 +294,13 @@ void testFluentdQuery() {
     }
 
     logger.info("Total count of logs: " + count);
-    assertTrue(count > 0, "Total count of logs should be more than 0!");
-    assertEquals(0, failedCount, "Total failed count should be 0!");
     logger.info("Total failed count: " + failedCount);
 
-    logger.info("Query logs of serverName={0} succeeded", adminServerPodName);
+    if (count > 0 && failedCount == 0) {
```

Comment:
You can simplify this to just return the test: `return count > 0 && failedCount == 0;`

Reply:
changed